autopush-common/src/db/bigtable/bigtable_client/error.rs

```diff
@@ -75,41 +77,40 @@ impl ToString for MutateRowStatus {
             MutateRowStatus::Unavailable => "Unavailable",
             MutateRowStatus::DataLoss => "Data Loss",
             MutateRowStatus::Unauthenticated => "Unauthenticated",
-        }
-        .to_owned()
+        })
     }
 }
 
 #[derive(Debug, Error)]
 pub enum BigTableError {
-    #[error("Invalid Row Response")]
-    InvalidRowResponse(grpcio::Error),
+    #[error("Invalid Row Response: {0}")]
```

Comment:
Won't these produce errors that are harder to collect in Sentry? 
The argument is being displayed in the format and as part of the event, so it's not getting lost.

Reply:
I don't think this really complicates Sentry's collection -- we do [the same in syncstorage](https://github.com/mozilla-services/syncstorage-rs/blob/cc8d5847dc37de321a484321b222e0757bc322ab/syncstorage-spanner/src/error.rs#L66)

Outside of Sentry this is helpful, e.g. when diagnosing a db error locally which you typically aren't using Sentry with.

Reply:
Fair.